scripts/deploy.js

```diff
@@ -360,6 +360,17 @@ async function ensureVerifierDeployment(plan, config) {
     }
 }
 
+async function ensureStateBridgeDeployment(plan, config) {
```

Comment:
This isn't ensuring the deployment of anything. You don't use `plan`, so you can just take `config` and instead call it `getStateBridgeAddress`.

Reply:
fixed